src/containers/career.jsx

```diff
@@ -24,14 +24,49 @@ class Career extends Component {
 
     this.state = {
       currentIndex: 0,
-      careers: []
+      careers: [
```

Comment:
in the future it's better to have all of your states in redux not in the component state.
but it's fine for now